scipy/stats/tests/test_qmc.py

```diff
@@ -534,6 +555,58 @@ def test_sample_stratified(self):
         assert np.any(sample - expected > 0.5 / n)
 
 
+class TestOptimalLatinHypercube(QMCEngineTests):
+    qmce = partial(qmc.OptimalLatinHypercube, n_perturbations=100)
+    can_scramble = False
+    unscramble_nd = np.array([[0.60881992, 0.00416027],
+                              [0.82837347, 0.26284543],
+                              [0.20290629, 0.83297228],
+                              [0.48412877, 0.89496811],
+                              [0.06857794, 0.43212172],
+                              [0.98461223, 0.5690004],
+                              [0.3424405, 0.14251516],
+                              [0.64745145, 0.70599331]])
+
+    def test_continuing(self, *args):
+        pytest.skip("Not applicable: not a sequence.")
+
+    def test_fast_forward(self, *args):
+        pytest.skip("Not applicable: not a sequence.")
+
+    def test_discrepancy_hierarchy(self):
+        # base discrepancy as a reference for testing
+        # OptimalLatinHypercube is better
+        seed = np.random.RandomState(123456)
+        lhs = qmc.LatinHypercube(d=2, seed=seed)
+        sample_ref = lhs.random(n=20)
+        disc_ref = qmc.discrepancy(sample_ref)
+
+        # all defaults
```

Comment:
This doesn't seem to use all defaults. Do you mean e.g. `# using a random initial LHS`?

Reply:
I updated the tests. So closing this one as not relevant anymore.